### ISSUE TITLE
build.rs performance

### ISSUE THREAD

**Issue by dalance** [jsinger67/parol#49]
I found the following configuration improves `parol::builder::Builder` performance.
It enables optimization of build.rs. (By default build.rs seems to be debug mode)

```Cargo.toml
[profile.dev.build-override]
opt-level = 3

[profile.release.build-override]
opt-level = 3
```

Before: `parol build time: 9350 milliseconds`

After: `parol build time: 1273 milliseconds`

So the suggestion of the configuration may be good in the parol book.

**Comment by jsinger67** [comment]
Hi @dalance,
good point. Thank you for reporting this. 👍 
I will add the appropriate information to the parol book.
Maybe this should be added to the template for Cargo.toml when parol generates a new user project (parol new).

**Comment by jsinger67** [comment]
Fixed with [d10c783](https://github.com/jsinger67/parol/commit/d10c7835905cb17eab273ccac56ebb2329014f10)
The book is updated too.
Have fun! 🎉

**Comment by ryo33** [comment]
I want to include it in `parol new` command's output. I will send a pull request when #50 is done.
